packages/uswds-core/src/styles/settings/_settings-components.scss

```diff
@@ -213,6 +214,7 @@ $theme-table-sorted-header-background-color: "accent-cool-light" !default;
 $theme-table-sorted-background-color: "accent-cool-lighter" !default;
 $theme-table-sorted-stripe-background-color: "blue-cool-10v" !default;
 $theme-table-sorted-icon-color: default !default;
+$theme-table-sticky-top-offset: -1px !default;
```

Comment:
Small concern here is this is similar to the current hero setting where it's not very flexible or re-usable.

Not a blocker for this work.

Reply:
Yeah, the setting is fairly limited, but I am not sure if there is a way around it. I would imagine a lot of projects will need to write some custom style rules for the top offset.

Reply:
@amyleadem does this setting accept units too or only pixels? Do we have any guidance for this?